### DISCUSSION TITLE
I want to edit the source code of and rebuild a specific library. How can I achieve this?

### DISCUSSION THREAD

**Discussion by timluther** [microsoft/vcpkg#20716] [General]
Hi,
 I want to edit a line on a cpp file within a vcpkg project, then rebuild that project. This isn't for a permanent change, it's merely to test a theory so that I can give the developers a well informed ticket as to why their library now runs terribly slow in debug mode if that theory proves correct. 
 It sounds pretty straight forward - all the tools should be there to do this but it's not obvious. I can find the file in the 'buildtrees' directory but I can't see any commands to simply do what vcpkg does when it installed a triplet and build the thing. 
 Does anybody know?

**Comment by autoantwort** [comment]
I think [this](https://github.com/microsoft/vcpkg/blob/master/docs/examples/patching.md) is the way to go.
